src/time.js

```diff
@@ -28,13 +28,28 @@ const updateCityTileTime = (cityTimezone, itemName) => {
   }, 60000)
 }
 
+/**
+ * This function will get the current date
+ * @param {string} cityTimezone Timezone name
+ * @returns  {Array} returns the date array
+ */
+
+const updateDate = (cityTimezone) => {
```

Comment:
As this function returns and not updates
```suggestion
const getDate = (cityTimezone) => {
```